op-mode-definitions/ntp.xml.in

```diff
@@ -6,18 +6,30 @@
         <properties>
           <help>Show peer status of NTP daemon</help>
         </properties>
-        <command>${vyos_op_scripts_dir}/show_ntp.sh --sourcestats</command>
+        <command>${vyos_op_scripts_dir}/ntp.py show --command sourcestats</command>
         <children>
-          <node name="system">
+          <node name="activity">
+            <properties>
+              <help>Report the number of servers and peers that are online and offline</help>
+            </properties>
+            <command>${vyos_op_scripts_dir}/ntp.py show_summary --command activity</command>
```

Comment:
I don't think `activity`, `sources`, etc. should be options of `show`. There are a few factors:

* The set of those commands is finite, pre-defined, and very small.
* `vyos.opmode` will automatically pick up all `show_*` functions, so there's no big cost for adding commands, you can just do `def show_activity`, `def show_sources`...
* Commands defined as functions will automatically show up in the GraphQL API as well, and keywords are always better than string arguments that need to be validated and aren't a part of the schema.

Reply:
Okay, understood for show_activity and show_sources. Following this thinking, would you want to keep show and show_summary which calls "chronyc sourcestats -v" and "chronyc tracking" respectively or should adopt the approach using def show_chronycommand for all call consistency across the code with respect to chrony?

Reply:
We can and should keep the old behavior, it's just that the implementation will be different: here in `show ntp activity` the script call will be `ntp.py show_activity`, when there's a function `show_activity` in the script.

Reply:
@dmbaturin your feedback has been incorporated.